### DISCUSSION TITLE
Some problems about development of appium-doctor

### DISCUSSION THREAD

**Discussion by youngfreeFJS** [appium/appium#18294] [Q&A] [answered]
After appium-doctor going to be part of appium main repo, many files were deleted, such as babelconfig, gulpfile and some scripts in packjson.
I don't know how to develop debugging code under the doc package now.
Who can give some set up development examples? 🙏

**Comment by KazuCocoa** [comment, marked as answer]
```
npm install
npm run doctor
```

after code change:
```
npm run build
npm run doctor
```

Maybe these are the simplest way.

**Reply by youngfreeFJS** [reply]
Thanks. I forgot to read the scripts in appium package.json before.
